.github/workflows/pr_benchmarks.yml

```
@@ -0,0 +1,78 @@
+name: pr_benchmarks
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+        tags:
+          - '*'
```

```suggestion
    pull_request:
```
This one should only run on PRs, right?

---

Yes. Commiting this change.